core/migrations/0021_merge_20220331_0408.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 4.0.3 on 2022-03-31 04:08
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0019_pumping'),
+        ('core', '0020_bmi_tags_diaperchange_tags_feeding_tags_and_more'),
+    ]
+
+    operations = [
+    ]
```

Comment:
Hm how did we end up with this migration? It's not doing anything, right?

Reply:
There was a conflict probably due to the file names (two containing 019?) when I had to merge in from master. I can just rename my file if it makes it easier, though. This was what gulp generated when trying to merge the migrations.

Reply:
Ah ok -- yeah lets just have your migration recreated so we can ditch this empty one.

Reply:
fixed & fixed lint issues as well.